core/src/main/java/com/rudderstack/android/sdk/core/RudderConfig.java

```diff
@@ -627,11 +663,34 @@ public Builder withTrackLifecycleEvents(boolean shouldTrackLifecycleEvents) {
             return this;
         }
 
+        private boolean useNewLifecycleEvents = Constants.USE_NEW_LIFECYCLE_EVENTS;
+
+        /**
+         * @param shouldUseNewLifecycleEvents Whether we should use new lifecycle events
+         * @return RudderConfig.Builder
+         */
+
+        public Builder withUseNewLifecycleEvents(boolean shouldUseNewLifecycleEvents) {
```

Comment:
by convention, this should be either `useNewLifecycleEvents` or `withNewLifecycleEvents`

Reply:
changed it to `withNewLifeCycleEvents`